src/core_plugins/kibana/public/management/sections/indices/add_data_steps/parse_csv_step/parse_csv_step.js

```diff
@@ -21,6 +22,8 @@ modules.get('apps/management')
         const maxSampleRows = 10;
         const maxSampleColumns = 20;
 
+        this.maxBytesFormatted = numeral(addDataMaxBytes).format('0 b');
```

Comment:
It's used in [the view](https://github.com/elastic/kibana/pull/8228/files/0503aa8a31979870c196ce4ae96c4974f8c5b9eb#diff-57a75d69f6f81554cd117991f6f3f272R12)